nav2_bringup/bringup/params/nav2_multirobot_params_1.yaml

```diff
@@ -135,15 +136,17 @@ local_costmap:
     ros__parameters:
       use_sim_time: True
       global_frame: odom
-      plugin_names: ["obstacle_layer", "inflation_layer"]
-      plugin_types: ["nav2_costmap_2d::ObstacleLayer", "nav2_costmap_2d::InflationLayer"]
       rolling_window: true
       width: 3
       height: 3
       resolution: 0.05
       robot_radius: 0.22
-      inflation_layer.cost_scaling_factor: 3.0
+      plugins: ["obstacle_layer", "inflation_layer"]
+      inflation_layer:
```

Comment:
Add these changes in parsing / namespacing to the migration guide.

Foxy has all our deps now released so we need to release a foxy branch soon, so this will likely be one of the last things to go into it